<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>memberAccountView</class>
 <widget class="QMainWindow" name="memberAccountView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMenuBar {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                      stop:0 lightgray, stop:1 darkgray);
    spacing: 3px; /* spacing between menu bar items */
}

QMenuBar::item {
    padding: 1px 4px;
    background: transparent;
    border-radius: 4px;
}

QMenuBar::item:selected { /* when selected using mouse or keyboard */
    background: #a8a8a8;
}

QMenuBar::item:pressed {
    background: #888888;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget{
	background: #FFE5D0;
}</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>191</width>
      <height>471</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <widget class="QLabel" name="loanedBooksIcon">
       <property name="styleSheet">
        <string notr="true">QLabel{
	color: rgb(231, 138, 107);
}</string>
       </property>
       <property name="text">
        <string>Catalogue</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loanedBooksBtn">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border-radius:15px;
	font: 14px &quot;Montserrat&quot;;
}

QPushButton:hover {
	text-decoration: underline;
}</string>
       </property>
       <property name="text">
        <string>Loaned Books</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="reservedBooksIcon">
       <property name="text">
        <string>member</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="reservedBooksBtn">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border-radius:15px;
	font: 14px &quot;Montserrat&quot;;
}

QPushButton:hover {
	text-decoration: underline;
}</string>
       </property>
       <property name="text">
        <string>Reserved Books</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="editAccountIcon">
       <property name="text">
        <string>back to catalogue</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editAccountBtn">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border-radius:15px;
	font: 14px &quot;Montserrat&quot;;
}

QPushButton:hover {
	text-decoration: underline;
}</string>
       </property>
       <property name="text">
        <string>Edit Account</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="bcktoCatologueIcon">
       <property name="text">
        <string>back to catalogue</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bcktoCatologueBtn">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border-radius:15px;
	font: 14px &quot;Montserrat&quot;;
}

QPushButton:hover {
	text-decoration: underline;
}</string>
       </property>
       <property name="text">
        <string>Back to Catologue</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="signoutIcon">
       <property name="text">
        <string>signOut</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="signoutBtn">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border-radius:15px;
	font: 14px &quot;Montserrat&quot;;
}

QPushButton:hover {
	text-decoration: underline;
}</string>
       </property>
       <property name="text">
        <string>Sign Out</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>21</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Line{
	color:rgb(221, 46, 68);
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="logoImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/resources/images/bblLogo.png</pixmap>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>170</y>
      <width>931</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{
	border-radius: 25px;
	background: rgb(255, 255, 255);
}

/*QLineEdit{
	background:rgb(255, 229, 208);
	border-radius: 15px;
}*/

QLabel{
	background: rgb(255, 255, 255);
	font-size: 14px;
	font-weight: bold;

}

QLineEdit {
	background:rgb(255, 229, 208);
	border: 1px solid;
	border-radius: 15px;
	border-color: #e5e5e5;
	background-color: palette(base);
	padding: 0px 5px;
}

QDateEdit{
	background:rgb(255, 229, 208);
	border: 1px solid;
	border-radius: 15px;
	border-color: #e5e5e5;
	background-color: palette(base);
	padding: 0px 5px;
}

QFormLayout{
	color: rgb(255, 255, 255);
}


QPushButton {
	border-radius: 15px; 
	font-size: 13px;
}
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>340</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
background-color: rgba(231, 138, 107, 0.5);
color:  #fff;
font-weight: 500;
}</string>
     </property>
     <property name="text">
      <string>Save Changes</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	font-weight: bold;
	font-size: 18px;

}</string>
     </property>
     <property name="text">
      <string>Log-in Details</string>
     </property>
    </widget>
    <widget class="QLabel" name="userName">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>69</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="memberUser">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>50</y>
       <width>551</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="pass">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>67</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="memberPass">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>90</y>
       <width>551</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="email">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>270</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>124</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>36</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="memberEmail">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>273</y>
       <width>551</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
    <widget class="QLabel" name="mobileNumber">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>190</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Mobile Number</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="memberMobile">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>190</y>
       <width>551</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="dob">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>230</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Date of Birth</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="memberDOB">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>230</y>
       <width>250</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="firstName">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>160</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Full Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="memberFirstName">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>150</y>
       <width>551</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="pageTitle">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font-size: 24px;
	font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string>Edit Account Details</string>
    </property>
   </widget>
   <widget class="QLabel" name="pageDesc">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>80</y>
      <width>971</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Hey User, change your details here.
if you have any complaints, please contact our admin.</string>
    </property>
   </widget>
  </widget>
  <action name="actionView_Catalogue">
   <property name="text">
    <string>View Catalogue</string>
   </property>
  </action>
  <action name="actionSign_Out">
   <property name="text">
    <string>Sign-Out</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
